application/dashboard/trello_service.py

```diff
@@ -71,17 +77,17 @@ def set_credentials(self, api_key, api_token):
     def get_measure_cards(self):
         cards = [card for card in self.client.get_board(BOARD_ID).all_cards() if card.closed is False]
 
-        card_dicts = [self.map_card(card) for card in cards]
```

Comment:
@TheDoubleK what's the rationale behind adding the `_`?  Presumably there's no conflict with a global `map_card` method?   Could it be double-underscored to make it 'private'?

Reply:
Single underscore conventionally indicates a private method in Python.  Double underscore makes it "really" private by garbling the name in the compiler so it's impossible to call it from outside the class.  

So with double underscore it wouldn't be possible to test the method individually (as I've done for `_remove_internal_reference`).  `map_card` and the other methods I've used single underscore for are definitely not things we should be calling from outside this class, but it seemed nice to be able to unit test them.

Ideally we wouldn't do that and would have a test setup that creates a realistic Trello API response with a bunch of cards that fully exercise the way we want things to work.  But that was beyond what I could face at this point.

Reply:
BTW I think the "conflict with a builtin name" convention was to have the underscore after the name, rather than before.

Reply:
@TheDoubleK ah, so they're private-but-not-really methods?

Personally, I'm not sure we should be testing those directly (but I can see the temptation).